docker-compose-tests.yml

```diff
@@ -0,0 +1,65 @@
+version: '2.1'
+
+services:
+
+  test-etl:
+    extends:
+      file: docker-compose.yml
+      service: base-etl
+    command: yarn test
+    volumes:
+      - pomi-etl-node-modules-test:/code/node_modules
+    environment:
+      NODE_ENV: test
+      SNYK_TOKEN: ${SNYK_TOKEN}
```

Comment:
This could go, if you agree on not running `snyk` as part of `test`. Comment has been added against `package.json`